src/components/NumberInput/NumberInput-test.js

```diff
@@ -203,18 +212,21 @@ describe('NumberInput', () => {
       it('should invoke onClick when up arrow is clicked', () => {
         upArrow.simulate('click');
         expect(onClick).toBeCalled();
+        expect(onClick).toHaveBeenCalledWith(expect.anything(), 'up');
```

Comment:
wasn't sure how to create a synthetic event to pass to expect so `.anything()` matches anything that is not undefined or null. It was called with undefined before